docs/current_docs/api/clients-cli.mdx

```diff
@@ -21,77 +25,117 @@ The `dagger core` command lets you call the core Dagger API, without needing to
       file --path="./hello" \
       export --path="./hello-from-dagger"
     ```
-
-- You want to experiment with Dagger without having to program a Dagger module.
-
-    ```shell
-    dagger core container from --address=alpine file --path=/etc/os-release contents
-    ```
-
-- You want a nice alternative to `docker run`.
-
-    ```shell
-    dagger core container from --address=cgr.dev/chainguard/wolfi-base terminal
-    ```
-
-## Custom Dagger functions
-
-The Dagger API can be extended with Dagger [modules](../features/modules.mdx). The `dagger call` command lets you call Dagger functions from Dagger modules, either remotely or after installing them locally with `dagger install`.
-
-- The `dagger call` command lets you invoke a Dagger Function.
-
-    Here is an example of using it to call one of the auto-generated Dagger Functions:
-
+    </TabItem>
+    <TabItem value="dagger core">
     ```shell
-    dagger call container-echo --string-arg="Welcome to Dagger!" stdout
+    dagger shell -c 'container \
+      | from cgr.dev/chainguard/wolfi-base \
+      | with-exec apk,add, go \
+      | with-directory /src https://github.com/golang/example#master \
+      | with-workdir /src/hello \
+      | with-exec go,build,-o,hello,. \
+      | file ./hello \
+      | export ./hello-from-dagger'
     ```
+    </TabItem>
+    </Tabs>
 
-    Here's what you should see:
-
-    ```shell
-    Welcome to Dagger!
-    ```
-
-    :::tip
-    When using `dagger call`, all names (functions, arguments, struct fields, etc) are converted into a shell-friendly "kebab-case" style.
-    :::
-
-- Modules don't need to be installed locally. Dagger lets you consume modules from GitHub repositories as well. You can use Dagger Functions from external modules in exactly the same way as you would use them locally.
-
-    Here is an example of calling a Dagger Function from an external module, which scans container images for vulnerabilities.
+- You want to experiment with Dagger without having to program a Dagger module.
 
+    <Tabs>
+    <TabItem value="dagger shell">
     ```shell
-    dagger -m github.com/jpadams/daggerverse/trivy@v0.5.0 call scan-image --image-ref=ubuntu:latest
+    dagger shell -c 'container | from alpine | file /etc/os-release | contents'
     ```
-
-    Here's what you should see:
-
+    </TabItem>
+    <TabItem value="dagger core">
     ```shell
-    ubuntu:latest (ubuntu 24.04)
-    ============================
-    Total: 10 (UNKNOWN: 0, LOW: 8, MEDIUM: 2, HIGH: 0, CRITICAL: 0)
-    ...
+    dagger core container from --address=alpine file --path=/etc/os-release contents
```

Comment:
Just fyi, the equivalent to `dagger core` is `dagger -n` (`-n` means `--no-mod`, while `-m` is for `--mod`).

By default `dagger` will try to load a module if you're in one, and fallback to just core. And even if a module is loaded, the core functions are also there. So what you have is correct, but if you know you only want core, adding `-n` will be faster because it skips the checking and potential loading of a module.

Reply:
https://github.com/dagger/dagger/issues/9854#issuecomment-2727691596

Reply:
We have 24 instances of `dagger core` in our docs (not counting the changes in this PR). 

Should we change all instances of `dagger core` to `dagger -n` and remove `dagger core` from our docs?